instrumentation/restlet/restlet-1.0/testing/src/main/groovy/io/opentelemetry/instrumentation/restlet/v1_0/AbstractRestletServerTest.groovy

```diff
@@ -156,6 +157,11 @@ abstract class AbstractRestletServerTest extends HttpServerTest<Server> {
     true
   }
 
+  @Override
+  boolean testErrorBody() {
+    false
+  }
+
```

Comment:
just curious what I missed in the PR that affected this?

Reply:
This isn't obvious. The original groovy code for this assert was
```groovy
    if (testErrorBody()) {
      response.contentUtf8() == ERROR.body
    }
```
which apparently doesn't really work. Due to `if` it should explicitly use `assert`. So it previously passed only because this assertion wasn't really verified.